src/SFA.DAS.Commitments.Support.SubSite/Views/Apprenticeships/Search.cshtml

```diff
@@ -0,0 +1,107 @@
+﻿@using SFA.DAS.Commitments.Support.SubSite.Models
+@using SFA.DAS.Commitments.Support.SubSite.Enums
+@model ApprenticeshipSearchQuery
+
+
+@{
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<section class="search no-search-term">
+    <div class="grid-row">
+        <div class="column-full">
+
+            @using (Html.BeginForm("Search", "Apprenticeships", FormMethod.Get, new { accept_charset = "utf-8", @class = "search-header search-header-2", role = "search", id = "searchForm" }))
```

Comment:
Is there a reason why this form is returned using GET?
It means the result can be cached, so the user could perform a search and retrieve out of date results.
This StackOverflow page discusses sending a body with a GET request (which is generally considered a bad idea)....
https://stackoverflow.com/questions/978061/http-get-with-request-body